integration-cli/docker_cli_build_test.go

```diff
@@ -4879,6 +4879,23 @@ func (s *DockerSuite) TestBuildNotVerboseSuccess(c *check.C) {
 
 }
 
+func (s *DockerSuite) TestBuildNotVerboseFailureWithNonExistImage(c *check.C) {
+	// This test makes sure that -q works correctly when build fails by
+	// comparing between the stderr output in quiet mode and in stdout
+	// and stderr output in verbose mode
+	testRequires(c, Network)
+	TestName := "quiet_build_not_exists_image"
+	BuildCmds := "FROM busybox11"
```

Comment:
Ah, yes, looks like it's copied from `TestBuildNotVerboseFailure` below, which was like that